cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split3/SpeechToTextSDKSuite.scala

```diff
@@ -29,7 +29,7 @@ trait SpeechToTextSDKSuiteBase extends TestBase with CognitiveKey with CustomSpe
   import spark.implicits._
 
   val region = "eastus"
-  lazy val resourcesDir = new File(new File(getClass.getResource("/").toURI).toString)
```

Comment:
>new File(new File(getClass.getResource("/").toURI).toString) [](http://example.com/codeflow?start=26&length=60)

This nested File structure was causing a test failure. Not sure why it didn't cause problems in python or dotnet.